bugsnag.go

```diff
@@ -127,24 +194,48 @@ func init() {
 		sourceRoot = filepath.Join(runtime.GOROOT(), "src") + "/"
 	}
 	Config.update(&Configuration{
-		APIKey:        "",
-		Endpoint:      "https://notify.bugsnag.com/",
-		Hostname:      "",
-		AppType:       "",
-		AppVersion:    "",
-		ReleaseStage:  "",
-		ParamsFilters: []string{"password", "secret"},
-		SourceRoot:    sourceRoot,
+		APIKey: "",
+		Endpoints: Endpoints{
+			Notify:   "https://notify.bugsnag.com",
+			Sessions: "https://sessions.bugsnag.com",
+		},
+		Hostname:            "",
+		AppType:             "",
+		AppVersion:          "",
+		AutoCaptureSessions: true,
+		ReleaseStage:        "",
+		ParamsFilters:       []string{"password", "secret", "authorization", "cookie"},
+		SourceRoot:          sourceRoot,
 		// * for app-engine
 		ProjectPackages:     []string{"main*"},
 		NotifyReleaseStages: nil,
 		Logger:              log.New(os.Stdout, log.Prefix(), log.Flags()),
 		PanicHandler:        defaultPanicHandler,
 		Transport:           http.DefaultTransport,
+
+		flushSessionsOnRepanic: true,
 	})
 
 	hostname, err := os.Hostname()
 	if err == nil {
 		Config.Hostname = hostname
 	}
 }
+
+func startSessionTracking() {
+	if sessionTracker == nil {
```

Comment:
Is there a potential for multiple sessionTrackers to be instantiated here if this method is called concurrently?

Reply:
Yeah, good catch. Ensured that startSessionTracking only gets called once.